src/Kafka/ManageKafkaPermissions/components/Cells.tsx

```diff
@@ -69,7 +69,7 @@ export const ResourceCell: VFC<ResourceCellProps> = ({
 
 export type PermissionOperationCellProps = {
   permission: AclPermissionType;
-  operation: AclOperation;
+  operation: AclOperation | AclOperation[];
```

Comment:
I think we should try to make it more simple, just have one type 2d array `AclOperation[]` for manual and shortcuts permissions.

Reply:
This was done because Review permissions able passes a string to display the operation here, while the assign permissions needs an array.
I kept the string so the review permissions work as they are